yt/frontends/enzo_e/data_structures.py

```diff
@@ -399,6 +399,11 @@ def _parse_parameter_file(self):
         fh = h5py.File(os.path.join(self.directory, fn0), "r")
         self.domain_left_edge = fh.attrs["lower"]
         self.domain_right_edge = fh.attrs["upper"]
+        if 'version' in fh.attrs:
+            version = fh.attrs.get('version').tobytes().decode('ascii')
+        else:
+            version = None # earliest recorded version is '0.9.0'
+        self.parameters["version_string"] = version
```

Comment:
mapping a key ending with `"_string"` to anything other than `str` (e.g. `None`) seems a poor choice. I'd suggest either changing the fall back value to something like `"unknown"`, or renaming the key to `"version"` to avoid suggesting a fixed type

Reply:
Good point. I fixed that